fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -169,6 +172,12 @@ public PlanFragment visitPhysicalOlapScan(
         TableRef ref = new TableRef(tableName, null, null);
         BaseTableRef tableRef = new BaseTableRef(ref, olapTable, tableName);
         tupleDescriptor.setRef(tableRef);
+        olapScanNode.setSelectedPartitionIds(physicalOlapScan.getSelectedPartitionId());
+        try {
+            olapScanNode.updateScanRangeInfoByNewMVSelector(physicalOlapScan.getSelectedIndexId(), false, "");
+        } catch (Exception e) {
+            LOG.warn("set failed", e);
```

Comment:
what happens when an exception is caught? compute wrong result?

Reply:
change to throw exception